src/crypto.ts

```diff
@@ -0,0 +1,17 @@
+import type { CryptoProvider, HashID } from './cryptoTypes.js'
+import { CryptoSjcl } from './cryptoSjcl.js'
+
+class CryptoImpl implements CryptoProvider {
+    constructor(public provider: CryptoProvider) {}
+
+    get Group() {
+        return this.provider.Group
+    }
+
+    hash(hashID: HashID, input: Uint8Array): Promise<Uint8Array> {
+        return this.provider.hash(hashID, input)
+    }
+}
+
+// There's already a Crypto global which makes auto imports fail
+export const Crypto = new CryptoImpl(CryptoSjcl)
```

Comment:
Crypto.provider is configurable 

And the code is updated to reference Crypto.Group and Crypto.hash directly, rather than reference it on Oprf.Crypto or Oprf.Group.. 

As a bonus it removes circular dependency issues with dleq* and Oprf